v2/interface.go

```diff
@@ -1,18 +1,11 @@
 package v2
 
 import (
-	"github.com/confluentinc/confluent-kafka-go/kafka"
-
 	"github.com/honestbank/kp/v2/internal/middleware"
 )
 
 type KafkaProcessor[MessageType any] interface {
```

Comment:
`KPProcessor` or just `Processor`? since it doesn't necessarily take messages from kafka now.